blocks/block-edit/index.js

```diff
@@ -36,7 +37,16 @@ export function BlockEdit( props ) {
 	// them preferencially as the render value for the block.
 	const Edit = blockType.edit || blockType.save;
 
-	return <Edit { ...props } className={ className } />;
+	// For backwards compatibility concerns adds a focus and setFocus prop
+	// These should be removed after some time (maybe when merging to Core)
```

Comment:
I like it 👍

Reply:
On a second thought, we'd have to proxify the props object to do so, which I think is not really possible:(

Reply:
Well I guess it's fine, it seems supported where we need it.

Reply:
> we'd have to proxify the props object

What do you mean by this? I was just thinking of the following pseudo-code:

```python
if not isSelected: focus = false
else:
  if browserHasProxySupport: focus = deprecatedEmptyObject
  else: focus = {}
```

Reply:
I was trying to find a way to proxy any access to `props.focus` (and not `focus.*`) because the main use-case for this prop is just checking truthiness to show the block toolbar/inspector

Reply:
Gotcha.